manage.py

```diff
@@ -14,6 +14,7 @@
 app = create_app(os.getenv('BOILERPLATE_ENV') or 'dev')
 app.register_blueprint(blueprint)
 app.app_context().push()
+app.config['ERROR_404_HELP'] = False
```

Comment:
Sert à quoi ?

Reply:
A éviter que tes messages de'erreurs soient complétés automatiquement.
Sans ça tu vas avoir ton message à toi: User not found. You have requested the URI ...but did you mean...
Alors que nous on veut: User not found.

Reply:
ok top